examples/modules/extractors/plot_supported_formats.py

```diff
@@ -0,0 +1,93 @@
+'''
+Supported File Formats
+========================
+
+Currently, we support many popular file formats for both raw and sorted extracellular datasets. Given the standardized,
+modular design of our recording and sorting extractors, adding new file formats is straightforward so we expect this
+list to grow in future versions.
+'''
+
+##############################################################################
+# Recording Data Formats
+# ----------------------
+#
+# For raw data formats, we currently support:
+#
+# * **Binary** - BinDatRecordingExtractor
+# * **Biocam HDF5** - BiocamRecordingExtractor
+# * **Experimental Directory Structure (Exdir)** - ExdirRecordingExtractor
+# * **Intan** - IntanRecordingExtractor
+# * **Klusta** - KlustaRecordingExtractor
+# * **MaxOne** - MaxOneRecordingExtractor
+# * **MCSH5** - MCSH5RecordingExtractor
+# * **MEArec** - MEArecRecordingExtractor
+# * **Mountainsort MDA** - MdaRecordingExtractor
+# * **Neurodata Without Borders** - NwbRecordingExtractor
+# * **Open Ephys** - OpenEphysRecordingExtractor
+# * **Phy/Kilosort** - PhyRecordingExtractor/KilosortRecordingExtractor
+# * **SpikeGLX** - SpikeGLXRecordingExtractor
+# * **Spyking Circus** - SpykingCircusRecordingExtractor
+
```

Comment:
I don't think having a hard list in example is a good idea...
I would put this somewhere esle in the doc. Not in gallery.